src/hooks/useUploadsManager/useUploadsManager.tsx

```diff
@@ -54,7 +54,16 @@ export const UploadManagerProvider: React.FC = ({ children }) => {
     },
     { skip: !uploadsState.length }
   )
-  const pendingNotificationsCounts = useRef({ uploading: 0, uploaded: 0 })
+
+  const pendingUploadingNotificationsCounts = useRef(0)
+  type AssetParentObjectID = string
+  const assetsNotificationsCount = useRef<{
```

Comment:
Also, it's possible for there to be a clash bettwen channel and video ID, I think we will need to include parent type in the key